web/ui/dashboard/src/app/private/pages/project/apps/app-details/app-details.component.ts

```diff
@@ -42,26 +41,27 @@ export class AppDetailsComponent implements OnInit {
 		public privateService: PrivateService
 	) {}
 
-	ngOnInit() {
-		this.getAppDetails(this.route.snapshot.params.id);
+	async ngOnInit() {
+		this.isLoadingAppDetails = true;
 		this.checkScreenSize();
+		await this.getAppDetails(this.route.snapshot.params.id);
```

Comment:
Is there a reason you're awaiting here?
Is there an action to be performed after?